pkg/cluster/nodes/create.go

```diff
@@ -135,6 +135,10 @@ func createNode(name, image, clusterLabel, role string, mounts []cri.Mount, extr
 		"--tmpfs", "/run", // systemd wants a writable /run
 		// some k8s things want /lib/modules
 		"-v", "/lib/modules:/lib/modules:ro",
+		// current calico config needs ipv6 enable
+		// run the node as dual stack to support all scenarios
+		"--sysctl", "net.ipv6.conf.all.disable_ipv6=0",
+		"--sysctl", "net.ipv6.conf.all.forwarding=1",
```

Comment:
this seems suspiciously likely to break things

Reply:
yeah, I reverted it